packages/nodes-base/nodes/Airtop/actions/session/terminate.operation.ts

```diff
@@ -0,0 +1,48 @@
+import {
+	NodeOperationError,
+	type IDataObject,
+	type IExecuteFunctions,
+	type INodeExecutionData,
+	type INodeProperties,
+} from 'n8n-workflow';
+
+import { validateAirtopApiResponse } from '../../GenericFunctions';
+import { apiRequest } from '../../transport';
+
+export const description: INodeProperties[] = [
+	{
+		displayName: 'Session ID',
+		name: 'sessionId',
+		type: 'string',
+		required: true,
+		default: '={{ $json["sessionId"] }}',
+		displayOptions: {
+			show: {
+				resource: ['session'],
+				operation: ['terminate'],
+			},
+		},
+		description: 'The ID of the session to terminate',
+	},
+];
+
+export async function execute(
+	this: IExecuteFunctions,
+	index: number,
+): Promise<INodeExecutionData[]> {
+	const sessionId = this.getNodeParameter('sessionId', index) as string;
+
+	// validate sessionId
+	if ((sessionId || '').trim() === '') {
```

Comment:
You should create a `validateSessionId` method on `GenericFunctions.ts` since you could use it here and on `packages/nodes-base/nodes/Airtop/actions/window/create.operation.ts`